website/content/en/highlights/2021-12-15-splunk-hec-improvements.md

```diff
@@ -0,0 +1,37 @@
+---
+date: "2021-12-15"
+title: "Splunk HEC Improvements"
+description: "Improved compatibility with Splunk HTTP Event Collector"
+authors: ["barieom"]
+pr_numbers: [10261, 10135]
+release: "0.19.0"
+hide_on_release_notes: false
+badges:
+  type: announcement
+---
+
+We're excited to share that we've made a couple improvements to Vector's support
+for Splunk HEC; Vector now supports Splunk HEC indexer acknowledgements and
+token passthrough routing.
+
+With the new support for Splunk HEC indexer acknowledgement, Vector now provides a
+better end-to-end acknowledgement to ensure that no data is lost. As you may be
+aware, Splunk HEC does not guarantee that a data is successfully written and saved
+if it receives a 200. Vector will now check [Splunk HEC indexer acknowledgements][Splunk indexer]
+protocol to verify that data has been successfully delivered. To learn more about
+how it works, check out the short explanation in the Vector [sinks reference page][indexer how it works].
+
+Vector now also supports new token passthrough routing. You can now use these tokens,
+which can be found in `splunk_hec` source requests, downstream `splunk_hec` sink
+requests. With the ability to record Splunk token that the events came in with, you
+can now use that token in the `splunk_hec` sink to forward logs. With this update,
```

Comment:
This seems slightly repetitive with the previous sentence. Perhaps combine them into something like: 
> "When `store_hec_token` is enabled in a `splunk_hec` source, tokens included in requests to the source will be stored and used by downstream `splunk_hec` sinks. Any passed through token takes precedence over the `default_token` configuration set in the sink." 

Side note: the token passthrough works for both `splunk_hec` metrics and logs sinks so "to foward logs" may be a bit confusing.